src/som/vm/Universe.java

```diff
@@ -53,21 +58,40 @@
 import som.vmobjects.SSymbol;
 
 
-public class Universe {
+public final class Universe {
 
-  public static void main(final String[] arguments) {
-    // Create Universe
-    Universe u = new Universe();
+  static final Layout LAYOUT = Layout.createLayout();
 
-    // Start interpretation
-    try {
-      u.interpret(arguments);
-    } catch (ProgramDefinitionError e) {
-      u.errorExit(e.toString());
-    }
+  private final TruffleLanguage.Env env;
+  private final GraalSOMLanguage language;
+  private final String testClass;
+  private final String testClasspath;
```

Comment:
Ok, so, I think this relates to the question of whether to allow multiple `eval()` calls from the launcher.
A Universe needs to initialize the object system, so, that takes loading and parsing various classes, which takes time. So, I'd try to not throw it away unnecessarily.